app.js

```diff
@@ -1,71 +1,99 @@
-function CreateBook(title, author) {
-  this.title = title;
-  this.author = author;
-}
-function remove(index) {
-  const booksArray = JSON.parse(localStorage.getItem('book'));
-  booksArray.splice(index, 1);
-  localStorage.setItem('book', JSON.stringify(booksArray));
-}
-const booksHolder = document.querySelector('.books-holder');
-function construc() {
-  let count = 0;
-  if (localStorage.getItem('book') != null) {
-    const storage = JSON.parse(localStorage.getItem('book'));
-    storage.forEach((element) => {
-      const itemTitle = element.title;
-      const itemAuthor = element.author;
-      const newUl = document.createElement('ul');
-      const liTitle = document.createElement('li');
-      liTitle.textContent = itemTitle;
-      newUl.appendChild(liTitle);
-      const liAuthor = document.createElement('li');
-      liAuthor.textContent = itemAuthor;
-      newUl.appendChild(liAuthor);
-      booksHolder.appendChild(newUl);
-      const newButton = document.createElement('button');
-      newButton.innerHTML = 'remove';
-      newButton.setAttribute('class', 'button-remove');
-      newButton.setAttribute('id', count);
-      booksHolder.appendChild(newButton);
-      const newHr = document.createElement('hr');
-      booksHolder.appendChild(newHr);
-      count += 1;
-    });
-    count = 0;
-  }
-  const removeButtons = document.querySelectorAll('.button-remove');
-  removeButtons.forEach((element) => {
-    element.addEventListener('click', (e) => {
-      remove(e.target.id, 1);
-      booksHolder.innerHTML = '';
-      construc();
-    });
-  });
-}
+/* eslint-disable max-classes-per-file */
```

Comment:
**First of all... a perfect implementation of refactoring in class.** 

[OPTIONAL] Second is optional, kindly remove eslint disabler for this file. 

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.

Reply:
Hello @Rustamxon7 thank you for the recommendation, how can we modify the .eslintrc.json file so it can ignore the rule of only one class per file?